.github/workflows/deploy-storybook.yml

```diff
@@ -42,9 +47,9 @@ jobs:
         run: yarn build-storybook
```

Comment:
I wonder if we need this step still? (And the `yarn install` above it too?)

Looks like this github action does both of those for us

Reply:
Agree on this feedback, not sure we need the steps Brandon noted. Can we trigger this build before merging it? I am not sure and I know that's not always possible with GHA.

Reply:
i dont think so aside from testing on your own repo

Reply:
Oh yeah, actually, i just duplicated the branch to our repo and it's now available in the Actions tab 
<img width="1645" alt="Screenshot 2024-03-22 at 11 46 18 AM" src="https://github.com/trussworks/react-uswds/assets/764090/e4073959-3ae7-4d53-ae4f-7ef602dae361">

Should I run it?

Reply:
@jpandersen87 it was easier for us to test and iterate on a Truss-owned branch. If you make your PR diff match https://github.com/trussworks/react-uswds/pull/2844/files, I'm happy to approve so you get the credit!

Thanks for taking the time! PS: Our latest storybook deploy is live thanks to this change: https://trussworks.github.io/react-uswds/

Reply:
haha, I was watching the deploy and saw the node16 warnings and made the action version update changes literally the same time as you did